programs/server/Server.cpp

```diff
@@ -1792,10 +1796,6 @@ try
                              "No servers started (add valid listen_host and 'tcp_port' or 'http_port' "
                              "to configuration file.)");
 
-#if USE_SSL
-        CertificateReloader::instance().tryLoad(config());
```

Comment:
Is it okay to skip this call here? It seems to do more than just loding on the first call

Reply:
Will it be thread-safe to call `CertificateReloader::init()` init later from `ConfigReloader`?

Reply:
Okay, let's try if it helps